server.js

```diff
@@ -1,25 +1,34 @@
 const net = require('net');
 const JsonSocket = require('json-socket');
+const Agenda = require('agenda');
+
+
+/**
+ * Initialising Agenda queue with local Mongo backed persistence,
+ * 50 default concurrent jobs and 70 max cuncurrent jobs
+ */
+const mongoConnectionString = "mongodb://127.0.0.1/queue";
```

Comment:
You should also define a collection name (https://github.com/rschmukler/agenda#databaseurl-collectionname), @triglian with one should they use?